editor/state.js

```diff
@@ -34,7 +40,20 @@ export const blocks = combineUndoableReducers( {
 
 		return state;
 	},
-	byUid( state = {}, action ) {
+
+	edits( state = {}, action ) {
+		switch ( action.type ) {
+			case 'EDIT_POST':
```

Comment:
When do we call `EDIT_POST` ? Or is the idea that we'll use it in follow-up pull requests? How does this relate to `UPDATE_BLOCK` (insert, remove)?